ansible/roles/debops.tinyproxy/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+- name: Install required APT packages
+  package:
+    name: '{{ q("flattened", tinyproxy__base_packages
+                             + tinyproxy__packages) }}'
+    state: 'present'
+  register: tinyproxy__register_packages
+  until: tinyproxy__register_packages is succeeded
+  notify: [ 'Restart tinyproxy' ]
+
+- name: Divert the original tinyproxy configuration file
+  command: dpkg-divert --quiet --local --divert /etc/tinyproxy/tinyproxy.conf.dpkg-divert --rename /etc/tinyproxy/tinyproxy.conf
+  args:
+    creates: '/etc/tinyproxy/tinyproxy.conf.dpkg-divert'
+
+- name: Generate tinyproxy configuration
+  template:
+    src: 'tinyproxy.conf.j2'
+    dest: '/etc/tinyproxy/tinyproxy.conf'
+    owner: 'root'
+    group: 'root'
```

Comment:
You should drop the `owner` and `group` parameters from tasks, as long as they default to `root`:`root`. This can improve support for other platforms that use different UNIX group for `root`, for example FreeBSD.